{"uuid":"9b4feae3-49fd-49e5-b669-4516eca22699","children":["129c1748-0207-4241-8d3f-9b48d4a94a00"],"befores":[{"status":"failed","statusDetails":{"message":"`cy.visit()` failed trying to load:\n\nhttps://folio-snapshot.dev.folio.org/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND folio-snapshot.dev.folio.org\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer","trace":"CypressError: `cy.visit()` failed trying to load:\n\nhttps://folio-snapshot.dev.folio.org/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: getaddrinfo ENOTFOUND folio-snapshot.dev.folio.org\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n    at https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158141:82\n    at visitFailedByErr (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:157494:12)\n    at https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158140:11\nFrom previous event:\n    at go (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158103:17)\n    at https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158169:20\nFrom previous event:\n    at visit (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158167:38)\n    at Context.visit (https://folio-snapshot.dev.folio.org/__cypress/runner/cypress_runner.js:158176:14)\nFrom Your Spec Code:\n    at Context.eval (https://folio-snapshot.dev.folio.org/__cypress/tests?p=cypress\\support\\index.js:2695:6)\n\nFrom Node.js Internals:\n  Error: getaddrinfo ENOTFOUND folio-snapshot.dev.folio.org\n      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26)\n  "},"stage":"finished","steps":[{"statusDetails":{},"stage":"pending","steps":[{"statusDetails":{},"stage":"pending","steps":[],"attachments":[],"parameters":[],"start":1643789914645,"name":"visit (\"/\")"}],"attachments":[],"parameters":[],"start":1643789914644,"name":"login (\"diku_admin\"; \"admin\")"}],"attachments":[],"parameters":[]}],"afters":[],"name":"ui-data-import: MARC file upload with the update of instance, holding, and items"}